### PR TITLE
Support cell deletion suggestions

### PR DESCRIPTION
**Pull request by trungleduc**
## User-facing changes
- The cell suggestion button is now a drop-down menu
- Add `Suggest change` and `Suggest delete` option 

https://github.com/user-attachments/assets/3d9a949c-b2f5-46b5-b816-a7f1df24de2c

### CODE REVIEW

**Comment by trungleduc** [comment]
This PR is built on top of #13

**Comment by trungleduc** [comment]
Rebased after #13

File: packages/base/src/types.ts [hunk 1]
```diff
@@ -197,6 +224,12 @@ export interface ISuggestionsModel extends IDisposable {
    * @returns The cell instance, or -1 if the cell is not found.
    */
   getActiveCell(): Cell<ICellModel> | null | undefined;
+
+  /**
+   * Scroll to the cell by its id
+   *
+   */
+  nativateToCell(cellId?: string): void;
```
Comment (jtpio) [inline packages/base/src/types.ts:232]: Navigate?

```suggestion
  navigateToCell(cellId?: string): void;
```

Comment (jtpio) [reply packages/base/src/types.ts:232]: Or:

```suggestion
  scrollToCell(cellId?: string): void;
```

Comment (trungleduc) [reply packages/base/src/types.ts]: Thanks!

**Review by jtpio — APPROVED** [review]
:rocket:

**Pull request merged**